game_starter.py

```diff
@@ -38,42 +40,52 @@ def choose_word(word_list):
 # end of helper code
 # -----------------------------------
 
+
 # Load the list of words into the variable word_list
 # so that it can be accessed from anywhere in the program
 word_list = load_words()
 
+
 def is_word_guessed(secret_word, letters_guessed):
     '''
     secret_word: string, the word the user is guessing
     letters_guessed: list, what letters have been guessed so far
     returns: boolean, True if all the letters of secret_word are in letters_guessed;
       False otherwise
     '''
-    # FILL IN YOUR CODE HERE...
-    pass
+    # Creates a set of letters that make up sekrit word
+    secret_letters = set([x for x in secret_word])
+
+    # checks if subset and returns
+    return secret_letters.issubset(set(letters_guessed))
 
 
-### Testcases
+# Testcases
 # print(is_word_guessed('apple', ['a', 'e', 'i', 'k', 'p', 'r', 's']))
 # print(is_word_guessed('durian', ['h', 'a', 'c', 'd', 'i', 'm', 'n', 'r', 't', 'u']))
 # print(is_word_guessed('pineapple', []))
 
 
-
 def get_guessed_word(secret_word, letters_guessed):
```

Comment:
When inserting underscores into your string, it's a good idea to add at least a space after each 
one, so it's clear to the user how many unguessed letters are left in the string (compare the 
readability of 
____
with 
_ _ _ _ 
).